tests/tasks/test_shell.py

```diff
@@ -91,9 +102,7 @@ def test_shell_initializes_and_runs_multiline_cmd():
         task = ShellTask()(command=cmd, env={key: "test" for key in "abcdefgh"})
     out = f.run()
     assert out.is_successful()
-    lines = out.result[task].result.decode().split("\n")
-    test_lines = [l for l in lines if l == "test"]
-    assert len(test_lines) == 8
+    assert out.result[task].result == "yes"
```

Comment:
For this test, since we are only returning the final line you might need to sort the environment prior to echo'ing in the `cmd` script

Reply:
change made via push